server-wildfly/src/test/java/org/jboss/tools/rsp/server/wildfly/test/beans/MockServerCreationUtilities.java

```diff
@@ -50,6 +49,7 @@
 	private static final String wildfly_11_0_jar = "wf11.0.0.mf.jboss-as-server.jar";
 	private static final String wildfly_12_0_jar = "wf12.0.0.mf.jboss-as-server.jar";
 	private static final String wildfly_13_0_jar = "wf13.0.0.mf.jboss-as-server.jar";
+	private static final String wildfly_14_0_jar = "wf14.0.0.mf.jboss-as-server.jar";
```

Comment:
Did you actually add this file? I can't tell by looking at the PR.

Reply:
added a comment as requested (wfl13 and wfl14 jars are missing and we should get rid of the jars at some point)